server/routes/journeys/manage/visit/handlers/visitTypeHandler.ts

```diff
@@ -0,0 +1,26 @@
+import { Request, Response } from 'express'
+import { Page } from '../../../../../services/auditService'
+import { PageHandler } from '../../../../interfaces/pageHandler'
+import OfficialVisitsService from '../../../../../services/officialVisitsService'
+import { refDataRadiosMapper } from '../../../../../utils/utils'
+
+export default class VisitTypeHandler implements PageHandler {
+  public PAGE_NAME = Page.VISIT_TYPE_PAGE
+
+  constructor(private readonly officialVisitsService: OfficialVisitsService) {}
+
+  public GET = async (req: Request, res: Response) => {
+    const visitTypes = await this.officialVisitsService.getReferenceData(res, 'VIS_TYPE_CODE')
+    res.render('pages/manage/visitType', {
```

Comment:
I'll sort the dodgy base data in the DB for these next. It was just a starter-for-ten for now, and will be refined by the migration from NOMIS, mapping NOMIS types (and statuses) to DPS types.